src/components/Common/Navigation/BreadcrumbWrap/BreadcrumbWrap.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * This function is used to render the breadcrumb for the Agent Issuer, Agent
+ * Holder, and Agent Verifier pages
+ * @returns A breadcrumb.
+ */
```

Comment:
Not the case anymore?

Reply:
This is why I don't like comments in the code, unless it's explaining the reason **why**